lib/rouge/lexers/csound/builtins.rb

```diff
@@ -0,0 +1,1639 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    class CsoundOrchestra
+      # Opcodes in Csound 6.09.0 at commit 73a8df5 from
+      #   csound --list-opcodes3
```

Comment:
Could you create a rake task that generates this file? (similar to `tasks/lua.rake` for example)